app/editor/src/components/tool-bar/styled/ToolBar.tsx

```diff
@@ -2,7 +2,10 @@ import styled from 'styled-components';
 import { Row } from 'tno-core';
 
 export const ToolBar = styled(Row)`
-  background-color: ${(props) => props.theme.css.backgroundColor};
+  background-color: ${(props) =>
```

Comment:
I'm not sure this really makes sense.  If they provide a className the styling for that classname should override the default behaviour on its own.  It shouldn't need logic here.  What you want should be more like a variant option.  Similar to many of our form components.

Reply:
Will adjust!